src/main/java/org/dcsa/ebl/controller/unofficial/ResetTransportDocumentController.java

```diff
@@ -0,0 +1,68 @@
+package org.dcsa.ebl.controller.unofficial;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.events.model.enums.ShipmentEventTypeCode;
+import org.dcsa.core.events.repository.BookingRepository;
+import org.dcsa.core.events.repository.TransportDocumentRepository;
+import org.dcsa.core.exception.ConcreteRequestErrorMessageException;
+import org.dcsa.ebl.repository.ShippingInstructionRepository;
+import org.dcsa.ebl.service.TransportDocumentService;
+import org.springframework.http.MediaType;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(
+    value = "unofficial/reset-transport-document",
+    produces = {MediaType.APPLICATION_JSON_VALUE})
+@Validated
+public class ResetTransportDocumentController {
+
+  private final TransportDocumentService transportDocumentService;
+  private final ShippingInstructionRepository shippingInstructionRepository;
+  private final TransportDocumentRepository transportDocumentRepository;
+  private final BookingRepository bookingRepository;
+
+  @PostMapping(path = "/{transportDocumentId}")
+  public Mono<Void> findById(
```

Comment:
findById is a strange method name for a void return type and an HTTP Post

Reply:
Renamed